resources/views/room.blade.php

```diff
@@ -14,9 +17,14 @@
             <h1>Seznam uporabnikov</h1>
         </div>
         <ul class="border border-white rounded-bottom rounded-top roomList" id="userList"></ul>
-        <div>
-            <a href="/" class="btn btn-outline-success mt-2">Generiraj</a>
-        </div>
+        <?php
+        if ($room->getAdminId()) {
+                echo
+                '<div>
+                    <a href="/user" class="btn btn-outline-success mt-2">Generiraj</a>
```

Comment:
Wouldn't it be better if the path to generate would be /generate instead of /user? :slightly_smiling_face: